<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>522</width>
    <height>360</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 233, 203);</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>521</width>
      <height>361</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0.443182, x2:1, y2:0.358, stop:0 rgba(19, 22, 24, 255), stop:0.425781 rgba(0, 27, 40, 255), stop:1 rgba(21, 27, 40, 255));</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="filename">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>521</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>13</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 233, 203);</string>
    </property>
    <property name="text">
     <string>Byke Living Legend.mp3</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignBottom|Qt::AlignHCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="artwork">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>80</y>
      <width>191</width>
      <height>191</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #2495FF;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>80</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sitka Display</family>
      <pointsize>16</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 233, 203);</string>
    </property>
    <property name="text">
     <string>Title</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>130</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sitka Display</family>
      <pointsize>16</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 233, 203);</string>
    </property>
    <property name="text">
     <string>Artist</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>190</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sitka Display</family>
      <pointsize>16</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 233, 203);</string>
    </property>
    <property name="text">
     <string>Album</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>250</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sitka Display</family>
      <pointsize>16</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 233, 203);</string>
    </property>
    <property name="text">
     <string>Genre</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="title">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>80</y>
      <width>235</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
border: 0;
border-bottom: 5px solid qlineargradient(spread:pad, x1:0.465, y1:0, x2:0.483925, y2:1, stop:0 rgba(5, 77, 0, 95), stop:0.34598 rgba(0, 247, 16, 255), stop:0.485998 rgba(255, 255, 255, 255), stop:0.566396 rgba(255, 255, 255, 255), stop:0.706414 rgba(0, 247, 16, 255), stop:1 rgba(5, 77, 0, 94));</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="artist">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>130</y>
      <width>235</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
border: 0;
border-bottom: 5px solid qlineargradient(spread:pad, x1:0.465, y1:0, x2:0.483925, y2:1, stop:0 rgba(5, 77, 0, 95), stop:0.34598 rgba(0, 247, 16, 255), stop:0.485998 rgba(255, 255, 255, 255), stop:0.566396 rgba(255, 255, 255, 255), stop:0.706414 rgba(0, 247, 16, 255), stop:1 rgba(5, 77, 0, 94));</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="album">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>190</y>
      <width>235</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
border: 0;
border-bottom: 5px solid qlineargradient(spread:pad, x1:0.465, y1:0, x2:0.483925, y2:1, stop:0 rgba(5, 77, 0, 95), stop:0.34598 rgba(0, 247, 16, 255), stop:0.485998 rgba(255, 255, 255, 255), stop:0.566396 rgba(255, 255, 255, 255), stop:0.706414 rgba(0, 247, 16, 255), stop:1 rgba(5, 77, 0, 94));</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="genre">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>250</y>
      <width>235</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
border: 0;
border-bottom: 5px solid qlineargradient(spread:pad, x1:0.476, y1:0, x2:0.483925, y2:1, stop:0 rgba(80, 0, 106, 179), stop:0.327039 rgba(247, 0, 255, 255), stop:0.418605 rgba(255, 226, 255, 255), stop:0.549834 rgba(255, 226, 255, 255), stop:0.638627 rgba(247, 0, 255, 255), stop:1 rgba(80, 0, 106, 178));</string>
    </property>
   </widget>
   <widget class="QPushButton" name="saveBtn">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>320</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI Semibold</family>
      <pointsize>15</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: #2495FF;
	border-radius: 20px;
}

QPushButton:hover{
	background-color: transparent;
	border: 1px solid #2495FF;
}

QPushButton:pressed{
	background-color: #21D84C;
}

</string>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QPushButton" name="closeBtn">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>0</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: transparent;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="metadata_qrc.qrc">
      <normaloff>:/Icons/res/Icons/x-circle.svg</normaloff>:/Icons/res/Icons/x-circle.svg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="minimizeBtn">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>0</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: transparent;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="metadata_qrc.qrc">
      <normaloff>:/Icons/res/Icons/minus.svg</normaloff>:/Icons/res/Icons/minus.svg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="message">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>5</y>
      <width>391</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Saving ...</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="metadata_qrc.qrc"/>
 </resources>
 <connections/>
</ui>
